data/json/recipes/food/brewing.json

```diff
@@ -0,0 +1,236 @@
+[
+    {
+        "type" : "recipe",
+        "result": "brew_milk_curdled",
+        "result_mult" : 18,
+        "category": "CC_FOOD",
+        "subcategory": "CSC_FOOD_OTHER",
+        "skill_used": "cooking",
+        "difficulty": 5,
+        "skills_required": [ "survival", 4 ],
+        "time": 30000,
+        "batch_time_factors": [50, 4],
+        "autolearn": true,
+        "book_learn": [["survival_book", 3] , ["textbook_survival", 3], ["manual_survival", 3]],
+        "components": [
+            [ [ "milk", 15 ] ],
+            [ [ "vinegar", 3 ] ],
+            [
+                [ "wild_herbs", 40 ],
+                [ "stomach_large", 1 ],
+                [ "hstomach_large", 1 ],
+                [ "stomach", 2 ],
+                [ "hstomach", 2 ]
+            ]
+        ]
+    },{
+        "type" : "recipe",
+        "result": "brew_whiskey",
+        "result_mult" : 3,
+        "category": "CC_FOOD",
+        "subcategory": "CSC_FOOD_DRINKS",
```

Comment:
It's difficult to find your recipes even knowing what to look for. These should move to `CSC_FOOD_BREW` along with all of the other related brewing/distilling recipes with the exception of those that belong in `CSC_CHEM_FUEL`